### PR TITLE
fix: add assertion for existence before continuing in legends.test

### PR DESCRIPTION
**Pull request by TCL735**
let's see if this works

### CODE REVIEW

File: cypress/e2e/shared/legends.test.ts [hunk 1]
```diff
@@ -500,9 +500,13 @@ describe('Legends', () => {
         .click()
       cy.getByTestID('add-resource-dropdown--new').click()
 
-      cy.getByTestID('page-title')
+      cy.getByTestID('page').should('exist')
```
Comment (genehynson) [inline cypress/e2e/shared/legends.test.ts:503]: wouldn't "page" exist on the dashboards list page too? Could we try and look for an element that's unique to the new dashboard page? Maybe like the "Add cell" button or something?

Comment (TCL735) [reply cypress/e2e/shared/legends.test.ts:503]: Strangely, in the dashboards list, the data-testid is "empty-dashboards-list" even when the list is not empty

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 56 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=1986)

File: cypress/e2e/shared/legends.test.ts (hunk 1 above)
Comment (TCL735) [reply cypress/e2e/shared/legends.test.ts]: @genehynson so i added a testID just for the dashboard page. That should make it unique.

**Review by genehynson — APPROVED** [review]

File: cypress/e2e/shared/legends.test.ts (hunk 1 above)
Comment (genehynson) [reply cypress/e2e/shared/legends.test.ts]: ah, awesome thanks!

**Pull request merged**